### PR TITLE
update deps

### PR DESCRIPTION
**Pull request by zarsky-broad**
read all changelogs and tested, nothing particularly interesting

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -6850,6 +6842,12 @@
             "p-limit": "^2.2.0"
           }
         },
+        "path-exists": {
```
Comment (breilly2) [inline package-lock.json:6845]: Is this a new transitive dependency? Also `paging-algorithm`

File: package-lock.json [hunk 2]
```diff
@@ -11838,13 +11845,29 @@
...
+      "version": "4.1.1",
+      "resolved": "https://registry.npmjs.org/react-transition-group/-/react-transition-group-4.1.1.tgz",
+      "integrity": "sha512-K/N1wqJ2GRP2yj3WBqEUYa0KV5fiaAWpUfU9SpHOHefeKvyrO+VrnMBML21M19QZoVbDZKmuQFHZYoMMi1xuJA==",
       "requires": {
+        "@babel/runtime": "^7.4.5",
         "dom-helpers": "^3.4.0",
         "loose-envify": "^1.4.0",
         "prop-types": "^15.6.2"
+      },
+      "dependencies": {
```
Comment (breilly2) [inline package-lock.json:11857]: This is new too. Not sure what it's about. Does this make sense to you?

File: package-lock.json (hunk 1 above)
Comment (zarsky-broad) [reply package-lock.json:6845]: I wouldn't worry about the changes in lower level deps in general. This one, for example, was already a transitive dep, it just changed where it was referenced. Paging-algorithm is a new algorithm used by our pagination package. As a rule, while it makes sense to sanity-check, we shouldn't get consumed by checking sub-deps, especially when we've read changelogs and tested.

File: package-lock.json (hunk 2 above)
Comment (zarsky-broad) [reply package-lock.json:11857]: This is an instance where one of our deps decided to declare an implicit dependency explicitly, to deal with a bug, and was expected. See above.

File: package.json [hunk 3]
```diff
@@ -69,11 +69,11 @@
     "eslint-plugin-lodash-fp": "^2.1.3",
     "eslint-plugin-react": "^7.13.0",
     "eslint-plugin-react-hooks": "^1.6.0",
-    "eslint-plugin-simple-import-sort": "^4.0.0-beta.1",
-    "husky": "^2.4.0",
+    "eslint-plugin-simple-import-sort": "^4.0.0",
```
Comment (zarsky-broad) [inline package.json:72]: 4.0.0 is identical to 4.0.0-beta.1

**Pull request merged**